### ISSUE TITLE
Issue while importing module

### ISSUE THREAD

**Issue by doshisunny** [dc-js/dc.leaflet.js#28]
I am having issue with importing dc.leaflet while importing it.  
`const dcLeaflet = require('dc.leaflet'); `
gives me `error unable to resolve path module`  
Currently I am able to resolve it by adding `index.js` in dc.leaflet inside node_module directory with following.
```
dc = require("dc");
leaflet = require('leaflet');
leaflet.markercluster = require('leaflet.markercluster');
crossfilter = require("crossfilter2");
module.exports = require("./dc.leaflet");
``` 
This is something we can not do with application deployment.
Do you have any alternative or better suggestion to so?

**Comment by gordonwoodhull** [comment]
Hi @doshisunny, thanks for the report. I'd be glad to add module support to dc.graph.js but I just haven't migrated to ES6 myself so I don't know what goes in the file.

Would you care to open a pull request? I could merge and do a new npm release fairly quickly.

**Comment by doshisunny** [comment]
@gordonwoodhull Thank you so much.
I created pull request.

**Comment by gordonwoodhull** [comment]
Thanks @doshisunny! Merged in <strike>0.3.2</strike> 0.3.3 and published to npm.
